### ISSUE TITLE
After server.stop, my client still able to connect to the server why ?

### ISSUE THREAD

**Issue by rameshrajagopal** [nghttp2/nghttp2#483]
Server side code:
```
  {  
    boost::posix_time::time_duration td = boost::posix_time::milliseconds(timeout);
    server.read_timeout(td);
    boost::system::error_code ec;
    server.listen_and_serve(ec, host.getName(), host.getPort(), true);
    server.join()
   }
Test Code Snippet:
 {
   1) stops the server 
   2) trying to connect to the server and expects the connection failure whereas it is getting session_connect_callback
  }
```
Any idea why?

**Comment by tatsuhiro-t** [comment]
Thank you for reporting this bug.  Previously, we did not close acceptor.
Fix committed via 7ca2660a29c1b69dea461c5020c537f9e30cc285

**Comment by tatsuhiro-t** [comment]
Closing since fixed version has been released.
